### PR TITLE
Fix `make docker-serve`

### LINKED ISSUE #150: Running `make docker-serve` fails
Running `make docker-serve` fails with only Docker installed and running:

```plain
$ make docker-serve
docker run --rm -it -v /Users/chuck/src/chuckwondo/carpentries-geospatial-python:/srv/jekyll -p 127.0.0.1:4000:4000 jekyll/jekyll:3.8.5 make serve
bundle exec jekyll serve
Traceback (most recent call last):
        11: from /usr/local/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        10: from /usr/local/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         9: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/setup.rb:20:in `<top (required)>'
         8: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler.rb:107:in `setup'
         7: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/runtime.rb:20:in `setup'
         6: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/runtime.rb:108:in `block in definition_method'
         5: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/definition.rb:226:in `requested_specs'
         4: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/definition.rb:237:in `specs_for'
         3: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/definition.rb:170:in `specs'
         2: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/spec_set.rb:81:in `materialize'
         1: from /usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/spec_set.rb:81:in `map!'
/usr/local/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/spec_set.rb:87:in `block in materialize': Could not find concurrent-ruby-1.2.2 in any of the sources (Bundler::GemNotFound)
make: *** [Makefile:26: serve] Error 1
make: *** [docker-serve] Error 2
```

Based upon these [detailed setup instructions](https://carpentries.github.io/lesson-example/setup.html), there appear to be a number of prerequisites (beyond simply having Docker installed and running) for the command to succeed. However, requiring installation of anything other than Docker seems to render the whole point of using Docker moot, which is precisely to avoid the need to install anything else locally.

### PR DESCRIPTION
**Pull request by chuckwondo**
Make `make docker-serve` work when only Docker is installed (and
running), and add basic instructions to `CONTRIBUTING.md` for viewing
changes locally.

Also, fix broken "a later episode" link in
`_episodes/01-intro-raster-data.md`.

Fixes #150

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,16 +109,31 @@ Each lesson has two maintainers who review issues and pull requests or encourage
 The maintainers are community volunteers and have final say over what gets merged into the lesson.
 To use the web interface for contributing to a lesson:
 
-1.  Fork the originating repository to your GitHub profile.
-2.  Within your version of the forked repository, move to the `gh-pages` branch and
+1. Fork the originating repository to your GitHub profile.
```
Comment (rbavery) [inline CONTRIBUTING.md:112]: @chuckwondo can you fix these bullet points so they are numbered again?

**Review by rbavery — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (rbavery) [reply CONTRIBUTING.md:112]: oh cool! I didn't know that. good as is then.

Comment (chuckwondo) [reply CONTRIBUTING.md:112]: Cool. Anything else?

**Review by rbavery — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (rbavery) [reply CONTRIBUTING.md:112]: LGTM!

**Pull request merged**